### PR TITLE
cleanup rpc flags

### PR DESCRIPTION
**Pull request by artkaseman**
The RPC flags were inconsistent, so I defaulted to no external RPC, and RPC methods = Auto (the default) which will be safe if externally expose and unsafe if not.  

We can't create a template that is good for everyone, so a default base secure template will be best, and we can help out RPC providers if they need additional help.

### CODE REVIEW

File: node-operators/networks/full-node.md [hunk 1]
```diff
@@ -119,7 +118,6 @@ Now, execute the docker run command. Note that you have to:
     purestake/moonbeam:{{ networks.moonbase.parachain_docker_tag }} \
     --ws-external \
     --rpc-external \
```
Comment (JoshOrndorff) [inline node-operators/networks/full-node.md:120]: You're recommending these for MacOS but not for Ubuntu.

File: node-operators/networks/full-node.md [hunk 2]
```diff
@@ -159,7 +156,6 @@ Now, execute the docker run command. Note that you have to:
...
```
Comment (JoshOrndorff) [inline node-operators/networks/full-node.md:158]: Same for collators

File: node-operators/networks/full-node.md [hunk 3]
```diff
@@ -293,10 +289,6 @@ The next step is to create the systemd configuration file. Note that you have to
          --ws-port {{ networks.parachain.ws }} \
          --pruning=archive \
          --state-cache-size 1 \
-         --unsafe-rpc-external \
-         --unsafe-ws-external \
-         --rpc-methods=Safe \
-         --rpc-cors all \
          --log rpc=info \
```
Comment (JoshOrndorff) [inline node-operators/networks/full-node.md:292]: This isn't a security concern, but why have collators log rpc stuff?

File: node-operators/networks/full-node.md (hunk 1 above)
Comment (albertov19) [reply node-operators/networks/full-node.md:120]: Yes. My understanding is that network="host" doesn't work on mac so you have to provide these extra flags.

Comment (artkaseman) [reply node-operators/networks/full-node.md:120]: good point, fixed.

File: node-operators/networks/full-node.md (hunk 2 above)
Comment (artkaseman) [reply node-operators/networks/full-node.md:158]: good point, fixed.

File: node-operators/networks/full-node.md (hunk 3 above)
Comment (artkaseman) [reply node-operators/networks/full-node.md:292]: fixed

File: node-operators/networks/full-node.md (hunk 1 above)
Comment (artkaseman) [reply node-operators/networks/full-node.md]: it was the --ws-external and --rpc external.  i left the host flags

**Pull request merged**